background.js

```diff
@@ -59,6 +58,24 @@ function getRandomInt(min, max) {
     return Math.floor(Math.random() * (max - min) + min); 
 }
 
+// get an array that holds selected number of index for contents array
+// to avoid the duplicate of contents
```

Comment:
This description is difficult to understand. Add a description of why indexes should be selected without duplication.